Sources/Hyperspace/Service/Transport/TransportService.swift

```diff
@@ -44,7 +44,7 @@ extension TransportService: Transporting {
             self?.handle(data: data, response: response, error: error, for: HTTP.Request(urlRequest: request), completion: completion)
         }
 
-        queue.async { [weak self] in
+        queue.sync { [weak self] in
```

Comment:
This was breaking a test, so in the interest of not making sweeping changes in what is essentially a CI PR, I made this asynchronous call synchronous. @GrandLarseny .